collections/_article/year-in-review-2024.md

```diff
@@ -0,0 +1,233 @@
+---
+title: "2024 in Review"
+excerpt: "This year was almost as packed as the 4.3 release, so let's go through the months and engine areas together and talk highlights."
+categories: ["newsletter"]
+author: Nat
+image: 
+date: 2024-12-31 10:00:00
+---
+ 
+And with that, the 2024 season comes to an end. What a year it has been, with the number of Godot submissions in gamejams rising to new heights, [more people working for the Godot Foundation than ever before](https://godotengine.org/article/godot-foundation-update-dec-2024/), and most industry award shows featuring one if not more games #MadeWithGodot. 
+
+So let us celebrate the past year by looking back at one key highlight per month - an event the team attended, a major project milestone being reached, or a fun moment online the Godot community gathered around. Likewise, we looked at the release dates of notable [Godot games on Steam](https://steamdb.info/tech/Engine/Godot/) and decided to introduce you to some favorites alongside our own news. 
+
+Of course, we cannot forget the sheer amount of technological progress being made by thousands of contributors, so for the purpose of this review we checked in with the different maintainers as well. Find a summary of the most notable change per engine area below (and tell us about your personal highlight [in the comments](https://forum.godotengine.org/))!
+
+## January
+<img alt="the commit that open-sourced Godot" src="/storage/blog/2024-review/commit.jpg"/>
+
+Nostalgia became the theme of January, as we compiled a small [game showcase](https://x.com/godotengine/status/1754601111229497650) leading up to Godot’s 10-year-anniversary (counting from the moment the engine was first open sourced). For every year from 2014 to 2024 we shared two gems found on Steam and itch.io with the community, celebrating an ever growing list of Godot releases since the beginnings of the project. 
+
+Have a look at that [monumental commit](https://github.com/godotengine/godot/commit/0b806ee0fc9097fa7bda7ac0109191c9c5e0a1ac) all those years ago!
+
+<hr>
+[KinitoPET](https://store.steampowered.com/app/2075070/KinitoPET)
+
+Desktop pets have risen to new popularity in 2024. *KinitoPET* combines this trend with the evergreen horror-genre to bring you a unique experience about virtual assistants.
+<hr>
+
+## February
+As part of our social media expansion, we also created an [*_official* Godot Engine account](https://www.twitch.tv/godotengine_official) on the streaming platform Twitch. 
```

Comment:
```suggestion
As part of our social media expansion, we also created an [*official* Godot Engine account](https://www.twitch.tv/godotengine_official) on the streaming platform Twitch. 
```
I assume that's not intended.

Reply:
it is intended - hence the cursive
just a bad pun, really